Corona-Warn-App/build.gradle

```diff
@@ -329,10 +329,6 @@ dependencies {
     kaptTest "com.google.dagger:dagger-android-processor:$dagger_version"
     kaptAndroidTest "com.google.dagger:dagger-android-processor:$dagger_version"
 
-    def assisted_injection_version = "0.6.0"
```

Comment:
@AlexanderAlferov do we have something to do if we remove a dependency?

Reply:
Good question I removed it from oss file. https://github.com/corona-warn-app/cwa-app-android/pull/2181/commits/83ea5cab1bc20aa5315a92f01137aa52726535f8

Reply:
@kaddaSz is aware of this and our pipeline should handle it.